packages/reporter/src/attempts/attempt-model.ts

```diff
@@ -194,9 +194,23 @@ export default class Attempt {
     }
   }
 
-  @action finish (props: UpdatableTestProps) {
+  @action finish (props: UpdatableTestProps, isInteractive: boolean) {
     this.update(props)
     this.isActive = false
+
+    // if the test is not open and we aren't in interactive mode, clear out the attempt details
+    if (!this.test.isOpen && !isInteractive) {
```

Comment:
Should this be verifying if the attempt was successful before clearing?

Reply:
Those checks are made in [test.isOpen](https://github.com/cypress-io/cypress/blob/53db0ea71f756cec93efd8cd0c95218aac9b893a/packages/reporter/src/test/test-model.ts#L81).